src/components/fx/layout_attributes.js

```diff
@@ -161,6 +175,7 @@ module.exports = {
     },
     selectdirection: {
         valType: 'enumerated',
+        role: 'info',
```

Comment:
Please remove all hardcoded `role: 'info'` and `role: 'style'` keys above as they no longer needed in v2. #5425

Reply:
This comment is not resolved. Please remove all `role` keys added here.

Reply:
Done